Multisig/UI/Dapps/DappsDataSource.swift

```diff
@@ -20,7 +20,7 @@ class DappsDataSource {
 
     init() {
         guard let chain = try? Safe.getSelected()?.chain else {
```

Comment:
@sche @elgatovital please bear in mind that we're not always will get a safe if the safe selection changes. If the data is cleared, then selection is changed to nil. This is an expected case and it should be handled.

This situation arised when this screen was still refreshing due to slow network. Edge case, but still 🤷